### DISCUSSION TITLE
Move/Resize Window Snap to Grid

### DISCUSSION THREAD

**Discussion by bsehovac** [kasper/phoenix#341] [Ideas]
Could it be possible to implement something like, when resizing/moving window, it displays some grid as visual indicator, and after action is complete, to snap to that grid, or to snap to it while moving? There are some window managers which let you select on the grid, and then resize window to it, but it would be cool if we could do it directly with windows...

**Comment by kasper** [comment]
@bsehovac Hey! So basically an ability to render visual cues about grids on the screen?
